api/app/wildfire_one.py

```diff
@@ -481,6 +529,52 @@ async def get_hourly_actuals_all_stations(
     return hourly_actuals
 
 
+async def get_wfwx_stations_from_station_codes(session, header, station_codes: Optional[List[int]]):
+    """ Return the WFWX station ids from WFWX API given a list of station codes. """
+    kamloops_station_codes = (322, 239, 1108, 305, 1082, 266, 346, 286, 344,
```

Comment:
I don't know about this. I think we should be honouring this list of station codes sent from the front end - then we can get rid of this hard coded list entirely.

Reply:
I should have added a comment here -- if we do not limit the stations, WFWX breaks (it returns a broken JSON response if you request too many stations). I think it should be addressed when we want to open it up to more than Kamloops.

Reply:
Oh - but then we should just get the list from the database - we have table planning_weather_stations - we can just take that instead of hard coding.

Reply:
> Oh - but then we should just get the list from the database - we have table planning_weather_stations - we can just take that instead of hard coding.

I agree, this change was made when they were in a JSON file that was more work to massage into a list of station codes.